### DISCUSSION TITLE
API Token for Pushover notifications

### DISCUSSION THREAD

**Discussion by ShanePharmD** [martomi/chiadog#233] [Q&A] [answered]
I am uncertain what API Token to use for the pushover notifications.  If someone could explain that I would appreciate it!

**Comment by martomi** [comment, marked as answer]
At the bottom of Pushover's dashboard, there's `Create an Application/API Token` link. You can create your own token from there - just give it any name you like.

**Reply by ShanePharmD** [reply]
Thank you!
